src/libraries/System.Numerics.Vectors/tests/Matrix4x4OrientationAwareTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Numerics.Tests
+{
+    public abstract class Matrix4x4OrientationAwareTests
+    {
+        protected static Matrix4x4 RightHandToLeftHand =
```

Comment:
Should this rather just be called `SwapHandedness` or similar instead, rather than having two APIs that are identical?

In general the premise is just that we're inverting which direction the `Z-axis` is oriented

Reply:
Yeah, this approach is more general in cases where the basis changes aren't  self-inverses but we only care about LH/RH so I changed to your suggestion.